sidechain/storage/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "its-storage"
+version = "0.8.0"
+authors = ["Integritee AG <hello@integritee.network>"]
+edition = "2018"
+
+[dependencies]
+codec = { package = "parity-scale-codec", version = "2.0.0", features = ["derive"] }
+log = "0.4"
+parking_lot = "0.11.1"
+rocksdb = "0.17.0"
+thiserror = "1.0"
+
+# local
+its-primitives = { path = "../primitives" }
+
```

Comment:
Judging from the error in the build, you probably need a `[dev-dependencies] mockall = { ... }` for the code to compile.